### ISSUE TITLE
is there any option not to cache the image to the nodes but to a common place like a statefulet?

### ISSUE THREAD

**Issue by sangamgo** [senthilrch/kube-fledged#196]

**Comment by roee-landesman** [comment]
My understanding of Kubefledged is that it is purpose-built to cache the images on the cluster worker nodes directly in order to increase image pull times from the pods. If all the images were stored in a central repository then pods on nodes not co-scheduled with that repository would need to download the images across a network which defeats the purpose of the controller. That said, there are centralized self-hosted repositories such as [Harbor](https://goharbor.io/) that you may find useful.

**Comment by senthilrch** [comment]
Yes kube-fledged's purpose is to allow caching images directly on the worker nodes. There are other projects that support the use case mentioned by @sangamgo
